src/main/resources/odf1.3/OpenDocument-v1.3-part2-packages_odt/content.xml

```diff
@@ -755,13 +755,13 @@
    <text:p text:style-name="Text_20_body">The OpenDocument specification defines conformance for packages, package consumers, and package producers, with two conformance classes called conforming and extended conforming. This chapter defines the basic requirements for the individual conformance targets.</text:p>
    <text:h text:style-name="Heading_20_2" text:outline-level="2"><text:bookmark-start text:name="__RefHeading__752789_826425813"/>Package Conformance<text:bookmark-end text:name="__RefHeading__752789_826425813"/></text:h>
    <text:h text:style-name="Heading_20_3" text:outline-level="3"><text:bookmark-start text:name="__RefHeading__752791_826425813"/><text:reference-mark-start text:name="OpenDocument Package"/>OpenDocument Package<text:bookmark-end text:name="__RefHeading__752791_826425813"/><text:reference-mark-end text:name="OpenDocument Package"/></text:h>
-   <text:p text:style-name="Text_20_body">An <text:span text:style-name="Def">OpenDocument Package</text:span> <text:span text:style-name="ISO_20_Keyword">shall</text:span> meet the following requirements:</text:p>
+   <text:p text:style-name="Text_20_body">An <text:span text:style-name="Def">OpenDocument Package</text:span> shall meet the following requirements:</text:p>
    <text:list xml:id="list294792977" text:style-name="Document_20_Conformance">
     <text:list-item>
-     <text:p text:style-name="P25">It <text:span text:style-name="ISO_20_Keyword">shall </text:span>be a Zip file, as defined by <text:reference-ref text:reference-format="text" text:ref-name="nref-zip">[ZIP]</text:reference-ref>. All files contained in the Zip file shall be non compressed (STORED) or compressed using the “deflate” (DEFLATED) algorithm.</text:p>
+     <text:p text:style-name="P25">It <text:span text:style-name="ISO_20_Keyword">shall</text:span> be a Zip file, as defined by <text:reference-ref text:reference-format="text" text:ref-name="nref-zip">[ZIP]</text:reference-ref>. All files contained in the Zip file <text:span text:style-name="ISO_20_Keyword">shall</text:span> be non compressed (STORED) or compressed using the “deflate” (DEFLATED) algorithm.</text:p>
     </text:list-item>
     <text:list-item>
-     <text:p text:style-name="P25">It <text:span text:style-name="ISO_20_Keyword">shall</text:span> contain a file “META-INF/manifest.xml”. This file <text:span text:style-name="ISO_20_Keyword">shall</text:span> meet the following requirements:</text:p>
```

Comment:
and here

Reply:
Yes, there where three occurrences all the same wording. Now all fixed.